src/order.c

```diff
@@ -2681,6 +2681,36 @@ void chr_order_chunk(bool decreasing,
   );
 }
 
+
+struct chr_order_data_exec {
```

Comment:
Unimportant: I'd call these `struct chr_order_info` and `struct chr_order_cleanup_info`.